Source/WarpX.cpp

```diff
@@ -554,11 +554,17 @@ WarpX::ReadParameters ()
 
         // Only needs to be set with WARPX_DIM_RZ, otherwise defaults to 1
         pp.query("n_rz_azimuthal_modes", n_rz_azimuthal_modes);
-#if (defined WARPX_DIM_RZ) && (defined WARPX_USE_PSATD)
+#if (defined WARPX_DIM_RZ)
```

Comment:
I made this same change in the PR1115. I would prefer the separate PR since it is not directly a part of this PR.

Reply:
Oh of course, sorry @dpgrote !